specs/LocaleRegion.md

```diff
@@ -0,0 +1,111 @@
+LocaleRegion
+===
+
+# Background
+Developers have requested access to programmatically choose the specific locale/region to use in WebView2. 
+The locale/region updates the JavaScript region (objects in the ECMA standard) which gets reflected as
+variations in time and date formats. Currently, the locale is by default set to the same value as the 
+display language. 
+
+You can use the `LocaleRegion` API to update the locale value individually from the display
+language.
+
+# Description
+* The intended value for `LocaleRegion` is in the format of `language[-country]` where `language` is the 
+2-letter code from [ISO 639](https://www.iso.org/iso-639-language-codes.html) and `country` is the 

```

Comment:
This is very specific, mentioning 2-letter codes from two standards. But BCP-47 supports a much richer set of codes (including 3-letter codes introduced in 2001). Is the precise definition here correct, or should it be relaxed to match BCP-47? (Even if you don't support all the features of BCP 47)

Reply:
Additionally, on Windows (and other systems) there are locales available that are more than just "language-script-region" codes.  You can have "neutrals" (no region, eg: "ja" is sufficient for ja-JP and sometimes used in place of that).  You can have variants as well.

Note that Windows will endeavor to "construct" the best data possible from *ANY* locale tag, so don't over-validate.  Any well-formed locale name is permissible.  Overvalidation prohibits use of app-defined tags, forward looking tags, or tags renamed for geopolitical reasons.

Reply:
For WebView2 we are often (and in this case) creating APIs on top of the existing Edge/chromium code base. In this case these docs are attempting to describe the behavior of that existing code and this feature is not itself implementing any new validation. It may be that we're not accurately describing the existing code and we'll look into this.

Reply:
Looking at the existing Edge/chromium/ code base, we see the value gets converted to BCP-47, so I will change this to match that. 
Does it make sense to add this link for reference: https://www.techonthenet.com/js/language_tags.php
?